packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -1,12 +1,7 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import { BaseComponent, classNamesFunction, divProperties, getNativeProps, createRef, getRTL } from '../../Utilities';
-import {
-  IScrollablePane,
-  IScrollablePaneProps,
-  IScrollablePaneStyles,
-  IScrollablePaneStyleProps
-} from './ScrollablePane.types';
+import { IScrollablePane, IScrollablePaneProps, IScrollablePaneStyles, IScrollablePaneStyleProps } from './ScrollablePane.types';
```

Comment:
Many of these changes look to be from a Prettier configuration change, and aren't related to this issue.